src/calendar/calendar-day.style.tsx

```diff
@@ -0,0 +1,176 @@
+import { Color } from "../color";
+import { TextStyleHelper } from "../text";
+import styled, { css } from "styled-components";
+import { DayCellStyleProps, VariantDay } from "./calendar-day";
+
+interface DayNumberProps {
+    $variant: VariantDay;
+    $disabled: boolean;
+}
+
+interface GrowDayCellProps extends DayCellStyleProps {}
+
+interface CircleProps extends Pick<DayNumberProps, "$variant"> {
+    $position: "left" | "right";
+}
+
+interface InteractiveCircleProps extends Omit<DayNumberProps, "$variant"> {}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const WeekDaysContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin-bottom: 0.5rem;
+`;
+
+export const WeekCell = styled.div`
+    ${TextStyleHelper.getTextStyle("H6", "semibold")}
+    height: 1.625rem;
+    width: 4.875rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: ${Color.Neutral[1]};
+    pointer-events: none;
+    user-select: none;
+`;
+
+export const CalendarDaySection = styled.div`
+    display: flex;
+    justify-content: space-around;
+    margin-bottom: 0.25rem;
+`;
+
+export const GrowDayCell = styled.div<GrowDayCellProps>`
+    display: flex;
+    position: relative;
+    height: 2.5rem;
+    align-items: center;
+    justify-content: center;
+
+    ${(props) => {
+        const { $selected } = props;
+
+        if ($selected) {
+            return css`
+                ${LeftCircle} {
+                    background-color: ${Color.Accent.Light[5]};
+                    border: 1px solid ${Color.Primary};
+                }
+
+                ${RightCircle} {
+                    background-color: ${Color.Accent.Light[5]};
+                    border: 1px solid ${Color.Primary};
+                }
+            `;
+        }
+    }}
+`;
+
+const DayCellBasic = styled.div`
+    position: relative;
+    height: 100%;
+    display: flex;
+    overflow: hidden;
+    width: 1.25rem;
+`;
+
+const Circle = styled.div<CircleProps>`
+    ${TextStyleHelper.getTextStyle("H5", "regular")}
+    color: ${Color.Neutral[1]};
```

Comment:
is color needed? seems to be handled in `DayNumber`

Reply:
Thank you so much for your time to understand my code what exactly does.